store/local_cache_supplier.go

```diff
@@ -33,6 +33,18 @@ type LocalCacheSupplier struct {
 	cluster       einterfaces.ClusterInterface
 }
 
+// Caching Interface
+type ObjectCache interface {
```

Comment:
It didn't make sense to define this in `lru.go` when it was only used here -- this is really a signature for the local cache supplier to define.

I intentionally didn't add the new methods added to the lru cache, since they aren't relevant to this supplier.